sovrin_client/test/cli/test_suspension.py

```diff
@@ -31,7 +32,7 @@ def tbgAdded(be, do, trusteeCli, nymAddedOut):
     vals = copy(vals)
     vals['target'] = vals['newTGBIdr']
     be(trusteeCli)
-    do('send NYM dest={newTGBIdr} role=TGB',
+    do('send NYM dest={{newTGBIdr}} role={role}'.format(role=Roles.TGB.name),
```

Comment:
Not sure about the TGB role. Is this a real role in our long-term vision, or is the TGB just empowered because the chair is Jason, who is a member of the Board of Directors and thus a Trustee?